cardillo/visualization/vtk_export.py

```diff
@@ -1,8 +1,10 @@
-import numpy as np
-from xml.dom import minidom
 from pathlib import Path
 from shutil import rmtree
+from xml.dom import minidom
+
```

Comment:
Can you remove the blank line here and below?

Reply:
👍